dataeng/resources/snowflake-replica-import.sh

```diff
@@ -2,18 +2,34 @@
 
 env
 
-${WORKSPACE}/analytics-configuration/automation/run-automated-task.sh \
- ImportMysqlDatabaseFromS3ToSnowflakeSchemaTask --local-scheduler \
- --date $(date +%Y-%m-%d -d "$RUN_DATE") \
- --credentials $SNOWFLAKE_CREDENTIALS \
- --warehouse $WAREHOUSE \
- --role $ROLE \
- --sf-database $SNOWFLAKE_DATABASE \
- --schema $SCHEMA \
- --scratch-schema $SCRATCH_SCHEMA \
- --run-id $BUILD_ID \
- --database $DATABASE \
- --overwrite \
- ${INCLUDE} \
- ${EXCLUDE} \
- ${EXTRA_ARGS}
+# Only invoke this script on Mondays, otherwise skip it.  Since this job is
+# triggered by an upstream for which we can't change the cron schedule (due to
+# vertica not yet being deprecated), we must skip it manually in this shell
+# script.
+
+DAY_OF_WEEK=$(date +%u)
+MONDAY=1
+if [[ ${DAY_OF_WEEK} -eq ${MONDAY} ]]; then
```

Comment:
I would suggest providing some flag that can override this logic, so that if we need to run this manually on a different day, we can do so.